tests/31sync/12receipts.pl

```diff
@@ -0,0 +1,110 @@
+test "Read receipts appear in initial v2 /sync",
+   requires => [qw( first_api_client can_sync )],
+
+   check => sub {
+      my ( $http ) = @_;
+
+      my ( $user, $filter_id, $room_id, $event_id );
+
+      my $filter = {
+         room => {
+            state => { types => [] },
+            timeline => { types => [] },
+            ephemeral => { types => ["m.receipt"] },
```

Comment:
Align these three lines' `=>` arrows.
Put some whitespace inside the `[ ]` brackets - `[ "m.receipt" ]`.